index.html

```diff
@@ -15,16 +15,171 @@
         <nav><i class="fa fa-bars"></i></nav>
     </header>
     <main>
+
+    <!-- THIS IS THE SECTION FOR MOBILE VERSION -->
+        <section class="grid-works">
+            <!-- CARD 1 (Tonic)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
+                    <h1 class="prjct-title">TONIC</h1>
```

Comment:
- [ ] Kindly change this `h1` tag to an `h2` tag. H1 tags are reserved to be used as a title to the website and should be used only once per page.